docs/1.getting-started/5.routing.md

```diff
@@ -132,7 +132,7 @@ If you have a more complex use case, then you can use anonymous route middleware
 <script setup>
 definePageMeta({
   validate: async (route) => {
-    const nuxtApp = useNuxtApp()
+    const route = useRoute()
```

Comment:
We can fully omit the line here as `route` is passed as argument already
```suggestion
```

Reply:
Thanks, I didn't notice that route is a parameter, because the previous part of the documentation mentions using the useRoute method to get the route object.

Reply:
Yes, that's correct. For `validate`, the route is given though so validation operations can be performed (as described in the doc text above the example) ☺️